modules/op-about-stepactions.adoc

```diff
@@ -0,0 +1,80 @@
+// This module is included in the following assemblies:
+// * about/understanding-openshift-pipelines.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-stepactions_{context}"]
+= StepAction
+
+A step is a part of a task. If you define a step in a task, you cannot reference this step from another task.
+
+However, optionally, you can define a `StepAction` custom resource (CR). This CR contains the action that a step performs. You can reference a `StepAction` object from a step to create a step that performs the action. You can also use resolvers to reference a `StepAction` object that is available in another namespace or from an external source.
```

Comment:
> You can also use resolvers to reference a `StepAction` object that is available in another namespace or from an external source.

Today ClusterResolver has no support for StepAction to resolve so imo this sentence `StepAction` object that is available in another namespace ` might create confusion :thinking: